src/parsec.rs

```diff
@@ -172,6 +172,19 @@ impl ParsecMap {
 
 /// Create Parsec instance.
 fn create(full_id: FullId, gen_pfx_info: &GenesisPfxInfo) -> Parsec {
+    use maidsafe_utilities::SeededRng;
+    use rand::Rng;
+    use unwrap::unwrap;
+
+    // Note: in production, use `OsRng` for maximum cryptographic security. In tests, however, this
+    // could cause "Too many open files" error because some tests create lot of nodes and each node
+    // has its own rng. To prevent that we use a weaker rng in tests.
+    let rng: Box<dyn Rng> = if cfg!(feature = "mock_base") {
+        Box::new(SeededRng::new())
```

Comment:
I'm not sure this is quite right, as this will create the same generator for all parsec instances.
Can we use another of the rand Rng?